package.json

```diff
@@ -41,10 +41,10 @@
     "sync-colors": "cd externals/elan && git fetch && git checkout origin/master && cp components/lib/variables/colors.json ../../data",
     "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/master && cp .env.example .env && yarn install && yarn dump-schema -- ../../data",
     "sync-schema:localhost": "cd ../metaphysics && yarn dump-schema -- ../reaction/data",
-    "test": "node verify-node-version.js && jest",
+    "test": "node verify-node-version.js && jest --no-cache",
```

Comment:
Should we keep `--no-cache` here by default or just leave it to devs to clear it on their own? Since we have so many tests these days the cache could help increase speed.

Reply:
I'll remove it, it was supposed to fix the decorators error but we can call it adhoc